MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -77,6 +77,26 @@ final class FeedUIIntegrationTests: XCTestCase {
 		assertThat(sut, isRendering: [image0])
 	}
 
+	func test_loadFeedCompletion_doesNotShowLoadFeedError() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoading()
+
+		XCTAssertFalse(sut.isShowingLoadingError)
+	}
+
+	func test_loadFeedError_tapToDismissLoadingError() {
+		let (sut, loader) = makeSUT()
+
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoadingWithError()
+		XCTAssertTrue(sut.isShowingLoadingError)
```

Comment:
To improve this test it's possible to check the error message being displayed, to ensure the `sut` is showing the correct error message.
```suggestion
		XCTAssertEqual(sut.errorMessage, localized("FEED_VIEW_CONNECTION_ERROR"))
```